ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -116,6 +135,8 @@ class AlertTabs extends Component {
           <AlertaConfig
             onSave={this.handleSaveConfig('alerta')}
             config={this.getSection(configSections, 'alerta')}
+            onTest={this.handleTestConfig('alerta')}
+            enabled={this.getEnabled(configSections, 'alerta')}
```

Comment:
`enabled` is calculated twice just a few lines apart (see line 134) and also is available in `config`- but.. both getting it from `config` in the component and putting the enabled status of each section seemed just as awkward as this. Suggestions would be welcome. Also keep in mind that this entire section is due for a refactoring, as issued here: (https://github.com/influxdata/chronograf/issues/2535)